### DISCUSSION TITLE
should i use this code in curretn century ? )

### DISCUSSION THREAD

**Discussion by workLes** [froii/testSlider#3] [Q&A] [answered]
should i use this code in curretn century ? )

**Comment by froii** [comment, marked as answer]
pff of course not - every year the capabilities of packages and libraries grow, so you need to use the latest version if possible!
if it's simple logic like this - you have to create it yourself using react hooks or whatever else you have in your time!
if it`s hard - you can find perfect code in github or npm ! 
good luck!
